src/main/webapp/js/instructor.js

```diff
@@ -434,3 +434,35 @@ function updateHoverShowPictureEvents(actualLink, resolvedLink) {
 }
 
 // --------------------------------------------------------------------------
+
+/**
+ * Selects contents inside an element.
+ * @param elementNode a HTML DOM element object, not a jQuery object
```

Comment:
Would it be good to follow the [coding standard](https://docs.google.com/document/d/1gZ6WG6HBTJYHAtVkz9kzi_SUuzfXqzO-SvFnLuag2xM/pub?embedded=true) (although it is deprecated) ?[https://github.com/airbnb/javascript/tree/es5-deprecated#comments](https://github.com/airbnb/javascript/tree/es5-deprecated#comments)
In this case, this will be changed to:
``` javascript
/**
 * Selects contents inside an element.
 * 
 * @param {HTML DOM object} elementNode
 */
```